include/db/sstable_files.hh

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include <cstddef>
+#include <cstdint>
+#include <string>
+#include <vector>
+
+#include "common.hh"
+
+namespace dibibase::db {
+
+enum FileType {
+  dataFile,
+  indexFile,
+  indexSummaryFile,
+  compressionInformationFile
+};
+
+class DIBIBASE_PUBLIC SSTableFiles {
+public:
+  explicit SSTableFiles(char *data, uint8_t sstable_num);
+  std::string get_file_name(uint8_t file_num);
```

Comment:
`get_file_name` is called in a loop, so I pass the current index to it, if I passed the FileType, I need to make switch case inside the loop, so if index = 0, I will pass the `DATA` to the `get_file_name`, instead of passing zero, and so on.
Is this better ? what do you think ?